core/chain-configs/src/client_config.rs

```diff
@@ -75,6 +75,8 @@ pub struct ClientConfig {
     pub tracked_shards: Vec<ShardId>,
     /// Not clear old data, set `true` for archive nodes.
     pub archive: bool,
+    /// Enable coloring of the logs
+    pub log_color: bool,
```

Comment:
I don't think we need to put it into chain config. The CLI argument should be enough. The same CLI argument should control all the colored output.

My reasoning is that the config is currently used to tweak long-term options while colors can be on in one context and off in another.

Reply:
So, the problem with using it as an argument we need to propagate it all way around through several strcutures just to tweak on if which is using it, and also having it in the config is kind of long term thing, as if you decide not to use color you dont want to pass the flag everytime to the binary.

Reply:
Well, I hear you. My point is that you usually want to run it without color in a supervisor, while you may want to run it with colors when you run it directly, and in this scenario, you will need to edit the config file just to tweak this.

I don't want us to spend more time on it than necessary, so I can agree on having it in the config. I would like to ask to have it next to the relevant `log_summary_period` configuration option, so we probably want to call it `log_summary_style` with string options of `"plain"` or `"colored"` (in the future we may introduce more of those, and I think it is easier to reason about the configuration option when it is not a boolean flag). To implement it, just introduce an enum with `#[derive(Serialize, Deserialize)]` and use it as a type of value.

Reply:
TIL some more rust, added the serde enum deserialization for the config like you suggested.